src/components/Dashboard/TodoList.vue

```diff
@@ -1326,8 +1329,11 @@ export default class TodoList extends Mixins(
           (item.correctedFilingType === FilingNames.CHANGE_OF_REGISTRATION) ||
           (item.correctedFilingType === FilingNames.REGISTRATION)
         ) {
-          // navigate to Edit UI to resume this correction
-          const correctionUrl = `${this.editUrl}${this.getIdentifier}/correction/?correction-id=${item.filingId}`
+          // navigate to Edit UI to resume this correction (depending on either CLIENT OR STAFF)
+          const userType = isEmpty(item?.type) ? '' : '?u=' + item?.type.toLowerCase()
```

Comment:
I think you don't need to pass the user type as a URL parameter -- it's already in the draft correction filing that the Edit UI will load.

Reply:
Sure. I'll remove the url for the Edit UI. I wasn't sure how Edit UI would load the correction.type.

Reply:
Let's discuss any time there is uncertainty so you don't code the wrong solution :)